python/_eos.cc

```diff
@@ -864,14 +864,23 @@ BOOST_PYTHON_MODULE(_eos)
         .def("__iter__", range(&SignalPDFs::begin, &SignalPDFs::end))
         ;
 
-    // }}}
+    // Analytic Charm Loops
+    def("F17_Qc",  (complex<double>(*)(const complex<double> &, const double &, const double &, const double &)) &agv_2019a::F17_Qc);
```

Comment:
OK as is for Javi's use. Before the next release, we should make sure that these free-standing functions are put into a dedicated class as static methods.

Reply:
My mistake!